CMakeLists.txt

```diff
@@ -583,7 +592,7 @@ if (ENABLE_KLEE_UCLIBC)
   endif()
 
   # Find the C library bitcode archive
-  set(KLEE_UCLIBC_BCA_NAME "klee-uclibc.bca")
+  set(KLEE_UCLIBC_BCA_NAME "klee-uclibc.bca") # For config.h
```

Comment:
Not used in code.

Reply:
Yes, it is: https://github.com/klee/klee/blob/ba0a6136cf6bd446af64e66d8b21e8ba1018a96e/tools/klee/main.cpp#L1114

The only thing that changed is that I changed passing this value via config.h rather than appending `-DKLEE_UCLIBC_BCA_NAME` to the `KLEE_COMPONENT_CXX_DEFINES`.